h/features.py

```diff
@@ -108,8 +142,8 @@ def features_status(request):
 
 
 def includeme(config):
+    # Remove old feature flags from the database on startup
+    config.action(None, remove_old_flags, order=90)
```

Comment:
Is there a central place that the order values for configuration actions could be defined, to make it more explicit what the groupings of actions are and in what order those groups run?